core/templates/components/question-directives/question-player/services/question-player-state.service.ts

```diff
@@ -62,7 +60,7 @@ export class QuestionPlayerStateService {
       linkedSkillIds: linkedSkillIds,
       answers: [],
       usedHints: [],
-      viewedSolution: null
+      viewedSolution: undefined
```

Comment:
```suggestion
```

Reply:
Made this change. Thanks!

Reply:
@vojtechjelinek You are suggesting to remove this? Removing this will lead to TS error of type mismatch of what constructor expects.